baitap_01/baitap_05.go/nhap.go

```diff
@@ -3,21 +3,18 @@ package main
 import "fmt"
 
 func main() {
+	const sotien_1_khach_chi = 120
 	S := 0
 	for i := 9; i < 21; i++ {
-		for 9 <= i < 11 {
-			fmt.Println("S", S+i*30*120)
+		if (i >= 9) && (i < 11) {
+			S += 30 * sotien_1_khach_chi
+		} else if (i >= 11) && (i < 13) {
+			S += 120 * sotien_1_khach_chi
+		} else if (i >= 13) && (i < 18) {
+			S += 40 * sotien_1_khach_chi
+		} else if (i >= 18) && (i < 21) {
+			S += 150 * sotien_1_khach_chi
 		}
-		for 11 <= i < 13 ;i++ {
-			fmt.Println("S", S+i*120*120)
-		}
-		for 13 <= i < 18 {
-			fmt.Println("S", S+i*40*120)
-		}
-		for 18 <= i < 21 {
-			fmt.Println("S", S+i*150*120)
-		}
-	} 
-	fmt.Println("S")
-
+	}
+	fmt.Println("Tổng doanh thu 1 ngày của nhà hàng là:", S, "nghìn vnđ")
```

Comment:
Hết vòng lặp ta có S: doanh thu 1 ngày của nhà hàng. Giờ chỉ cần in ra là kết thúc chương trình